common/lib/xmodule/xmodule/modulestore/tests/utils.py

```diff
@@ -255,19 +255,19 @@ def build(self, **kwargs):
         """
         contentstore = kwargs.pop('contentstore', None)
         if not contentstore:
-            with self.build_without_contentstore() as (contentstore, modulestore):
+            with self.build_without_contentstore(**kwargs) as (contentstore, modulestore):
```

Comment:
Some questions:

1. I see it being passed through the layers with `**kwargs`, but where does the `request_cache` actually get popped off and read?
2. Is it necessary to pass an explicitly created request cache all the way down? Could the the test case inherit from `CacheIsolationTestCase`, do its setup, and then test in uncached vs. cached states?